crates/core_arch/src/arm/neon.rs

```diff
@@ -1448,6 +1774,294 @@ mod tests {
         assert_eq!(r, e);
     }
 
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vand_s8() {
+        let a = i8x8::new(20, 20, 20, 20, 20, 20, 20, 20);
+        let b = i8x8::new(21, 21, 21, 21, 21, 21, 21, 21);
+        let e = i8x8::new(20, 20, 20, 20, 20, 20, 20, 20);
```

Comment:
Maybe vary the lane values a bit. That would test that the correct lane gets picked.

Reply:
makes a lot of sense!

Reply:
I change the data to cover the full spectrum.